plugins/workload-management/src/main/java/org/opensearch/plugin/wlm/rule/Rule.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.wlm.rule;
+
+import org.apache.commons.collections4.trie.PatriciaTrie;
+import org.opensearch.plugin.wlm.rule.storage.AttributeValueStore;
+import org.opensearch.plugin.wlm.rule.storage.TrieBasedStore;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * It is a stub which will be removed once Ruirui's PR for Rule schema is merged
+ */
+public class Rule {
```

Comment:
Assuming this class need not be reviewed as it is temporary stub

Reply:
yes